source/parser.hera

```diff
@@ -7989,22 +7990,34 @@ Init
             return
           }
 
-          let { pattern, block } = c
+          let { patterns, block } = c
+          let pattern = patterns[0]
```

Comment:
I think this definition of `pattern` isn't used. (I also can't figure out what it would be for... maybe indentation measuring?)

Reply:
Ohh, I see. Maybe worth adding a `testCase.skip` or adding to/opening new issue to document this. (In a perfect world, an error message would be useful, but that doesn't seem that important assuming we'll fix it soon enough.) ((Actually, how do you plan to transpile that...?))

Reply:
The current plan is:

```
if (Array.isArray(p) && p.length === 2) {
  const [x, y] = p;
  ...
else if (typeof p === "object" && p != null && "x" in p && "y" in p
) {
  const { x, y } = p;
  ...
}}
```

I'll add a test case.

Essentially adding a new condition and duplicating the block body with a different binding prefix.

Reply:
Oh nice, that seems like it would work well. A bit if a code parsing penalty from the code duplication, but at least it won't penalize network time much thanks to gzip.